jujugui/static/gui/src/app/assets/css/_mixins.scss

```diff
@@ -262,7 +262,8 @@
     color: $cool-grey;
     overflow-x: hidden;
     overflow-y: auto;
-    .spinner_loading {
```

Comment:
I was attempting BEM here, with "spinner" as the element and "loading" as the modifier, using a single underscore as a delimiter per https://en.bem.info/method/naming-convention/#modifier-name. I don't really care if we move away to something else, just wanted to explain the rational behind `spinner_loading`.